src/runtime/SrcDocIFrameController.ts

```diff
@@ -53,13 +70,30 @@ export class SrcDocIFrameController extends Disposable implements IEmbeddedConte
 
         await this.setRootClassNames(this.config.classNames);
         await this.setRootStyles(this.config.styles);
+        await this.addAnchorLinkHandlers();
```

Comment:
`addAnchorLinks` does not return a Promise, there's no need to `await` this call